src/datachain/data_storage/sqlite.py

```diff
@@ -333,16 +333,31 @@ def create_table(
         if_not_exists: bool = True,
         *,
         kind: str | None = None,
-    ) -> None:
-        self.execute(CreateTable(table, if_not_exists=if_not_exists))
+    ) -> bool:
+        """Create table and return True if created, False if already existed."""
+        table_existed = self.has_table(table.name)
+        if not table_existed or not if_not_exists:
```

Comment:
so, it will be default try to create it even if it already exists? according to this `if`